src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CreateCaseCallbackService.java

```diff
@@ -61,4 +82,26 @@ public CreateCaseCallbackService(
             .filter(item -> !Strings.isNullOrEmpty(item.getTransformationUrl()))
             .getOrElse(Validation.invalid("Transformation URL is not configured"));
     }
+
+    private Validation<Seq<String>, Map<String, Object>> createExceptionRecord(
+        ExceptionRecord exceptionRecord,
+        ServiceConfigItem configItem,
+        long caseId
+    ) {
+        try {
+            log.info("Start creating exception record for {} {}", configItem.getService(), caseId);
+
+            transformationClient.transformExceptionRecord(
+                configItem.getTransformationUrl(),
+                exceptionRecord,
+                "s2s token"
+            );
+
+            return Validation.valid(ImmutableMap.of("caseReference", UUID.randomUUID()));
```

Comment:
I assume this will change in the later prs?

Reply:
Yep, as soon as CCD API is included - the actual value is stuck in. Now just keeping the format so actually from this point endpoint can be developed. Quick note - will change the method signature to include idam token and user id coming from callback request. Not yet needed - not including

Reply:
A better name maybe should be validateExceptionRecordTransformation

Reply:
It would be best to get that dependency (CCD API stuff) in first.

Reply:
not really. ccd api dependency needs what's coming out from transformation api

Reply:
What I meant was that it's best to start with the deepest dependencies and move towards the root rather than the other way. The benefit is that whatever pieces of code we write, they're complete from the beginning and there's no need to "fix" them later (of course refactoring happens all the time). They can also be properly tested from the start.

This is kind of a general suggestion - I don't want to stop this piece of work if it's problematic to add that other thing first at this point.

Reply:
Then I guess blocking this ticket so validation can go in first was incorrect order, isn't it? From entry point of this ticket - validation was already present. Now going backwards ccd would need s2s and response from transformation api. Could, of course, just provide nulls or dummy things..

At this point it shouldn't really matter. No serious implications in code changes, service not in use, humanly you review a step-by-step approach.